documentation/release-process.md

```diff
@@ -5,7 +5,7 @@
 
 ## Prepare the release branch
 
-1. Update the [internal pipeline](https://dev.azure.com/dnceng/internal/_build?definitionId=954) variables to prevent consumption of nightly builds into dotnet-docker. Set `NightlyUpdateDockerFromMain` variable to `false` on the pipeline itself, not just on a new build.
+1. Update the [internal pipeline](https://dev.azure.com/dnceng/internal/_build?definitionId=954) variables to prevent consumption of nightly builds into dotnet-docker. Update `AutoUpdateDockerBranches` variable on the pipeline itself, not just on a new build, to the list of branch references (e.g. `refs/heads/main`) that you want to automatically update dotnet/dotnet-docker when scheduled builds are run; likely want to clear the variable when preparing for release.
```

Comment:
Let's include a sample of what `AutoUpdateDockerBranches` could look like in the docs here (e.g. `refs/heads/main;refs/heads/feature/9.x`)

Reply:
At this specific point in the instructions, you "likely want to clear the variable when preparing for release" (in the docs). Later, I give a specific example of what you want to set it back to.